package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/ccd-case-ui-toolkit",
-  "version": "1.2.27",
+  "version": "1.2.29",
```

Comment:
release new version as changes inside the library were needed as part of this PR (some fixes like missing resolver in CaseEditorModule)